test/test.hs

```diff
@@ -515,6 +515,21 @@ tests = testGroup "hevm"
         let e = BufLength (CopySlice (Lit 0x2) (Lit 0x2) (Lit 0x1) (ConcreteBuf "") (ConcreteBuf ""))
         b <- checkEquiv e (Expr.simplify e)
         assertBoolM "Simplifier failed" b
+    , test "simp-readByte" $ do
+      let srcOffset = (ReadWord (Lit 0x1) (AbstractBuf "stuff1"))
+          size =(ReadWord (Lit 0x1) (AbstractBuf "stuff2"))
+          src = (AbstractBuf "stuff2")
+          e = ReadByte (Lit 0x0) (CopySlice srcOffset (Lit 0x10) size src (AbstractBuf "dst"))
+          simp =Expr.simplify e
+      assertEqualM "readByte simplification" simp (ReadByte (Lit 0x0) (AbstractBuf "dst"))
+    , test "simp-readByte" $ do
+      let srcOffset = (ReadWord (Lit 0x1) (AbstractBuf "stuff1"))
+          size =(Lit 0x1)
+          src = (AbstractBuf "stuff2")
+          e = ReadByte (Lit 0x0) (CopySlice srcOffset (Lit 0x10) size src (AbstractBuf "dst"))
+          simp =Expr.simplify e
+      res <- checkEquiv e simp
```

Comment:
Wouldn't it be stronger to assert what the result of the simplification is?
I don't quite understand the purpose of this second test.

Reply:
I guess I wanted to prove that the simplification and the SMT solver agree... maybe I was a bit too eager to add 2 tests. It does prove that the SMT solver agrees, which is nice. Let me know what you think. Maybe it can be removed, honestly.

Reply:
Right, I see. I guess it is good to have a test for that as well!

Reply:
So, keep the test!
But please fix the spaces around `=` also in this test, as you've already done for the first one :)

Reply:
Oh man, I'm so sorry. I reviewed this PR and then I was like... let me add some tests. And then of course all the mess is from the tests... Sorry.